sydent/http/servlets/replication.py

```diff
@@ -1,106 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
renaming this file seems like a recipe for merge conflicts (and makes the review impossible)

please can you put it back. if you want to rename it, can you do it with a PR against master?